### DISCUSSION TITLE
Github Education plan duration

### DISCUSSION THREAD

**Discussion by Phoenix-rat** [community/community#38801] [GitHub Education] [answered]
Github Education has been going on throughout my student life. Do I need to submit documents again for the 2nd year? Also, will I be able to buy things like Microsoft Azure, Canva Pro for the second time in the 2nd year?

**Comment by grahamzemel** [comment, marked as answer]
Hey @Phoenix-rat! Great question, and the short answer is yes, you will have to submit documents periodically for verification purposes. However, upon submission each time, you'll continue to have access to the same resources you used before. 

This is so that Github isn't just handing out free benefits to 'students' who graduated college in 2000, and if they did it would likely shrink the benefits of Github Education. Hope that helps!

**Reply by Phoenix-rat** [reply]
Thx so much <3

**Reply by Phoenix-rat** [reply]
I have one more question @grahamzemel . My school does not give a new ID card every year. In the first year ( 2021-2022 ) an identity card was given once and we still continue on the same cards. Will it be a problem if I send this card again? Will it affect my approval?

**Reply by grahamzemel** [reply]
Nope, it should not be a problem. Great question though.

**Reply by samus-aran** [reply]
Awesome work @grahamzemel for assisting 👍 
Usually the duration for a valid student status is 2+ years. You'll be notified when it's time to reverify 👀
